recipes/heavydb/meta.yaml

```
@@ -0,0 +1,575 @@
+{% set version = "6.0.0" %}  # PEP 386
+{% set source_version = version %}
+{% set base_version = version %}
+{% set number = "3" %}
```

Bumping this again as we made change in the other repo.
```suggestion
{% set number = "4" %}
```